recipe/build.sh

```diff
@@ -14,10 +14,3 @@ cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
 make
 make install
 make test
-
-# hack so the old 1.4.6 .so still exists
-if [ `uname` == Darwin ]; then
-    ln -s $PREFIX/lib/libkea.1.4.7.dylib $PREFIX/lib/libkea.1.4.6.dylib
-else
-    ln -s $PREFIX/lib/libkea.so.1.4.7 $PREFIX/lib/libkea.so.1.4.6
-fi
```

Comment:
You might want to keep this in place until all the downstream projects start using this build number at build time.

Reply:
It's only `libgdal` and `rsgislib` that currently link against `kealib` and they are linked against 1.4.7. But you are right - it might break older builds...